docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -65,18 +76,16 @@ The following example shows how to create a gallery-like flexbox layout using th
     toolBar.JustifyContent = FlexContainer.Justification.JustifySpaceBetween;
     ```
 
-5.  Use the toolbar's `Flex` property to make the toolbar and the main content share the height of the main container, so that the toolbar occupies 10 percent of the vertical space and the content occupies the remainder:
+6.  Use `Flex` property of the toolbar to specify occupied area in main container by it. The main container's height is shared between toolbar and content (added in step 7). Since main container displays items vertically and toolbar `Flex` property is set to 0.1, toolbar will occupy 10% of verical and 100% of horizontal available space. Remainded area is intended for content:
 
-    ```
+    ```csharp
     /// Occupy 10 percent of available space on the cross axis
     toolBar.Flex = 0.1f;
     ```
 
-6.  To display the image, create the content area by adding a third flex container at the bottom of the main container.
-
-    Show the image in the center of the content area container by centering the item horizontally and vertically inside the area.
+7.  To display the image, create the content area by adding a third flex container at the bottom of the main container. Show the image in the center of the content area container by centering the item horizontally and vertically inside the area:
```

Comment:
```suggestion 
7.  To display the image, create the content area by adding a third flex container at the bottom of the main container. You can show the image in the center of the content area container by centering the item horizontally and vertically inside the area:

Reply:
@nawab-samsung How about removing redundancies? I think your "You can" can be omitted here.

Reply:
@mijin85cho  to better the sentence structure I used **you can**. 
Directly telling show doesnt go well with the sentence flow. 
If you do not agree, we can go with your suggestion . Kindly Let me know  your opinion,